pkg/render/manager.go

```diff
@@ -942,25 +944,27 @@ func managerUserSpecificSettingsGroup() *v3.UISettingsGroup {
 // all of the tigera namespaces.
 //
 // Calico Enterprise only
-func managerClusterWideTigeraLayer() *v3.UISettings {
+func managerClusterWideTigeraLayer(namespace []string) *v3.UISettings {
```

Comment:
nit: "namespace" is singular, but it's a slice of multiple elements.

This should be called something like `additionalNamespaces` or something like that. 

Alternatively, remove `namespaces` from this function altogether and callers just need to pass the full list of namespaces they need.

Reply:
Done. Passing the full list from the calling controllers.